client/src/components/game/lab2/components/header/headerMain.js

```diff
@@ -0,0 +1,131 @@
+import React, { Component } from "react";
+import ColorUpdate from "./buttons/colorUpdate";
+import Home from "./buttons/homeReset.js";
+import Button from "./buttons/button";
+import "../../Instructions/secondaryInstructions.css";
+
+/*
+Component for the header of the pages. Controls the buttons and options
+displayed to the users on each page of the appilcation
+*/
+class Header extends Component {
+
+  render() {
+    const {
+      gameState,
+      colors,
+      goBackFromGame,
+      changeGameColors,
+      aboutState,
+      closeAboutPage,
+      closeStatPage,
+      statState,
+      firstGame,
+      gamesPlayed,
+      openColorChange,
+      colorChange,
+      closeColorChange,
+      openSecondInfoState,
+      thirdInfoState,
+      gameMode,
+      endSystem
+    } = this.props;
+
+    const backButton = () => {
+      if (gamesPlayed === 2) {
+        closeColorChange();
+        openSecondInfoState();
+      } else {
+        changeGameColors(colors);
+        closeColorChange();
+      }
+    };
+    if (gameState) {
+      return (
+        <div className="headerStyle" style={{ background: "black" }}>
+          <div className="oneline center">
+            <Home
+              gameEnded={goBackFromGame}
+              colors={colors}
+              changeGameColors={changeGameColors}
+            />
+            <p className="deficiencyCheck">
+              Vision Deficiency Simulation:
+              <span style={{ fontWeight: "bold", marginLeft: "3px" }}>
+                {gameMode === "Main" ? " Off" : " On"}
+              </span>
+            </p>
+          </div>
+        </div>
+      );
+    }
+
+    return (
+      <div className="headerStyle">
+        <div>
+          {aboutState ? (
+            <Button
```

Comment:
the placement of the Back button is a little weird 
![image](https://user-images.githubusercontent.com/51053679/110518629-50b1fd80-80da-11eb-9c2c-5d3a2c796c6b.png)